src/main/java/crud/web/filters/AuthenticationFilter.java

```diff
@@ -0,0 +1,51 @@
+package crud.web.filters;
+
+import crud.lib.Injector;
+import crud.service.abstraction.DriverService;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AuthenticationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Injector injector = Injector.getInstance("crud");
+    private final Set<String> allowedUrls = new HashSet<>();
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
```

Comment:
```suggestion
    private final Set<String> allowedUrls = new HashSet<>();
```
we don't use it anymore